flume/doc/design/OrionTestSink.md

```diff
@@ -0,0 +1,23 @@
+# OrionTestSink
```

Comment:
Yes, at any time in the future I would like to make such a refactor, or at least, to document them with the same sections (functionallity, detailed configuration, uses cases).